decidim-core/app/cells/decidim/footer_topics_cell.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  # @todo Document this cell
+  #
+  # Example:
+  #
+  #    cell("decidim/footer_topics")
+  class FooterTopicsCell < Decidim::ViewModel
+    def show
+      return if topics.blank?
+
+      topics
+    end
+
+    private
+
+    def topics
+      @topics = current_organization.static_page_topics.where(show_in_footer: true)
```

Comment:
Missing memoization for the variable (see https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/). We also missed the actual creation of the array with the hashes that we consume in the view (using the `topic_item` method). This is what I come up with:

```suggestion
      @topics ||= current_organization.static_page_topics.where(show_in_footer: true) do |topic|
        {
          title: decidim_escape_translated(topic.title),
          path: decidim.page_path(topic.pages.first)
        }
      end
```


Mind that as there isn't any Topic page (i.e. this doesn't exist `decidim.topic_path(topic)`) , what we do is to link to the first page of the topic. 

I probably messed up the formatting here.